### PR TITLE
add ut for utils/http-middlewares

### PR DESCRIPTION
**Pull request by MregXN**
# Description

The test coverage of `dapr/utils/http-middlewares.go` is 0% currently. I added some simple scenarios to test its methods.

### CODE REVIEW

File: utils/http-middlewares_test.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUppercaseRequestMiddleware(t *testing.T) {
+	requestBody := "fake_body"
+	testRequest := httptest.NewRequest(http.MethodPost, "/test", strings.NewReader(requestBody))
+
+	var echoHandler = http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```
Comment (ItalyPaleAle) [inline utils/http-middlewares_test.go:30]: Linter

```suggestion
	echoHandler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

File: utils/http-middlewares_test.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+	expected := strings.ToUpper(requestBody)
+	assert.Equal(t, expected, rr.Body.String())
+}
+
+func TestResponseMiddleware(t *testing.T) {
+	testRequest := httptest.NewRequest(http.MethodGet, "/test", nil)
+
+	responseBody := "fake_body"
+	var responseHandler = http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```
Comment (ItalyPaleAle) [inline utils/http-middlewares_test.go:46]: Linter

```suggestion
	responseHandler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

**Review by ItalyPaleAle — APPROVED** [review]

File: utils/http-middlewares_test.go (hunk 1 above)
Comment (MregXN) [reply utils/http-middlewares_test.go]: modified, Thanks!

File: utils/http-middlewares_test.go (hunk 2 above)
Comment (MregXN) [reply utils/http-middlewares_test.go]: modified, Thanks!

**Review by yaron2 — APPROVED** [review]

**Review by mukundansundar — APPROVED** [review]

**Pull request merged**